clang/include/clang/Sema/Overload.h

```diff
@@ -1070,57 +1077,24 @@ class Sema;
     };
 
   private:
-    SmallVector<OverloadCandidate, 16> Candidates;
-    llvm::SmallPtrSet<uintptr_t, 16> Functions;
-
-    // Allocator for ConversionSequenceLists. We store the first few of these
-    // inline to avoid allocation for small sets.
-    llvm::BumpPtrAllocator SlabAllocator;
+    ASTContext &Ctx;
+    SmallVector<OverloadCandidate, 4> Candidates;
+    llvm::SmallPtrSet<uintptr_t, 4> Functions;
```

Comment:
pre-existing small nit, for consistency:
```suggestion
    SmallVector<OverloadCandidate, 4> Candidates;
    SmallPtrSet<uintptr_t, 4> Functions;
```

Reply:
The reason it works to drop the `llvm::` prefix for `SmallVector` is because we do this: https://github.com/llvm/llvm-project/blob/main/clang/include/clang/Basic/LLVM.h

`SmallPtrSet` is not included in there which is why the prefix is necessary, and I'm not certain it's worth including it or not.

Thoughts?

Reply:
I see, it's fine either way 😃